etc/systemd/bloop.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Bloop Scala build server
+
+[Service]
+ExecStart=%h/.bloop/bloop server
```

Comment:
Should we assume that `bloop` is in the `$PATH` and just use `bloop server` here?

I'm asking because we support `--destination` in the installation script, which will override the directory where Bloop is installed.

Reply:
Seems better to expect it to be on the path, otherwise we need to get the installation script to template this file with the chosen location (which sounds good as a solution too)

Reply:
True, I haven't noticed `--destination` when I started working on this. OTOH I don't think we can expect to have `bloop` in `$PATH` because systemd services don't use `.bashrc` etc., so a template seems to be the only solution. We'll also need templates for Desktop Entries anyway because we can't use environment variables at all there.

Reply:
Alternatively, we could try to add something like `BLOOP_PATH` to `~/.config/environment.d/`, which should be visible to the service and we could then use it in place of `%h/.bloop`. However, it's not that much different from using a template for `bloop.service`.

I don't think we should change `PATH` automatically because that's something that some users might not want to do.